test/run_clust.jl

```diff
@@ -0,0 +1,123 @@
+using ClustForOpt
+using Test
+using JLD2
+using Random
+# make sure to put Random.seed!() before every evaluation of run_clust, because most of them use random number generators in clustering.jl
+# Random.seed!() should give the same result even on different machines:https://discourse.julialang.org/t/is-rand-guaranteed-to-give-the-same-sequence-of-random-numbers-in-every-machine-with-the-same-seed/12344
+
+# put DAM_GER, DAM_CA, CEP GER1, CEP GER18 all years into data.jl file that just tests kmeans with all data inputs
+
+
+@load normpath(joinpath(dirname(@__FILE__),"reference_generation","run_clust.jld2")) reference_results
+
+Random.seed!(1111)
+@testset "run_clust $data" for data in ["CEP_GER1","CEP_GER18"] begin
```

Comment:
you could just use ``data in [:CEP_GER1, :CEP_GER18]``

Reply:
Good idea